lib/Clang.cpp

```diff
@@ -214,6 +214,33 @@ namespace pillar
 
     return clangVarDecl;
   }
+  clang::FieldDecl *ClangModuleImpl::createFieldDecl(clang::DeclContext *sdc, clang::DeclContext *ldc, clang::RecordDecl *record,
+                                                     const llvm::StringRef &name, const clang::QualType &type)
```

Comment:
take `llvm::StringRef` by value. It's already a lightweight wrapper pointing to an other-owned string.